### DISCUSSION TITLE
Disable evebox syslog

### DISCUSSION THREAD

**Discussion by grodish** [jasonish/evebox#269] [General]
Is there any way to disable evebox to write on syslog ?

Thanks

**Comment by jasonish** [comment]
EveBox doesn't write to syslog by itself, so I'm going to guess you mean the systemd journal? Info logging as default might be a little too verbose, and I see I have no way to limit it to warn or error only logging.  So I'll add that, probably a --quiet setting to disable info logs.

**Reply by JastrzebskiKrzysztof** [reply]
Good idea. rsyslog can be configured to "stop" but didn' found how get similar effect with systemd-journal.
"Events purged in last 60s" are logged every minute...

**Reply by jasonish** [reply]
I think this message can be removed.  If an issue is suspected, there's a more verbose way to look into it.  I first thought this might be a useful metric to have in logs by default, but I think I find it extraneous as well.

**Comment by skysor** [comment]
Yes please it would be great to remove them by default, or write them with a loglevel other than "warning"
right now they are writting 1 warning message each minute in my journald

**Comment by jasonish** [comment]
I've pushed these down to debug.  Without any `-v`, after the startup messages I think only warning and errors will now be logged.
